autoware_v2x_msgs/msg/VirtualGateCommand.msg

```diff
@@ -0,0 +1,3 @@
+builtin_interfaces/Time stamp
+autoware_v2x_msgs/VirtualGateAreaCommand[] areas
+autoware_v2x_msgs/VirtualGateVehicleInfo vehicle
```

Comment:
memo
will remove.
Because of the separate location information

Reply:
Removed https://github.com/autowarefoundation/autoware_msgs/pull/77/commits/3e7f9d553c5d777bd84eb523b696482d2c02254a.